tests/unit/classifier/svm/LibLinear_unittest.cc

```diff
@@ -6,7 +6,6 @@
  *
  * Written (W) 2014 pl8787
```

Comment:
while we are at it, could you make the license BSD? just copy it from anothoer file and maintain the old authors (add yourself)

Reply:
Done, changed it to BSD license.